src/pathfinding_service/api.py

```diff
@@ -468,8 +468,15 @@ def get(self, checksummed_address: str) -> Tuple[Dict[str, Union[str, PeerCapabi
                 UserPresence.ONLINE,
                 UserPresence.UNAVAILABLE,
             ]:
+                displayname = user_manager._displayname_cache.userid_to_displayname.get(
```

Comment:
What does it mean when there is no entry in `userid_to_displayname`? Did the user just not set a display name? For Raiden nodes, this must not be the case, right?

Reply:
This seems to be a rare case. This dictionary is populated [here](https://github.com/raiden-network/raiden/blob/develop/raiden/network/transport/matrix/utils.py#L142), and will remain empty in case this displayname is unknown to the Matrix server or if there was a communication problem.

Reply:
Furthermore, Presence is not stored if the displayname cannot be validated. It creates a new state called [`Presence.SERVER_ERROR`](https://github.com/fredo/raiden-services/blob/6e688c35503e8fc1998c1f2e4b405ea7916a81fa/src/raiden_libs/user_address.py#L316) which will show the node as offline from the outer perspective. 
As a consequence, any node which is seen as online, has displayname available.